packages/components/src/HeaderBar/HeaderBar.component.js

```diff
@@ -219,11 +237,14 @@ function HeaderBar(props) {
 			</ul>
 			<ul className={classNames(theme['tc-header-bar-actions'], theme.right)}>
 				{props.search && <Components.Search renderers={renderers} {...props.search} />}
-				{props.help && <Components.Help renderers={renderers} {...props.help} t={props.t} />}
-				{props.user && <Components.User renderers={renderers} {...props.user} />}
 				{props.notification && (
 					<Components.AppNotification renderers={renderers} {...props.notification} t={props.t} />
 				)}
+				{props.help && <Components.Help renderers={renderers} {...props.help} t={props.t} />}
+				{!props.user && (
```

Comment:
What do you think about usging function here instead?

Reply:
this seems quite straight forward to me and really quick, concise and easy to read